inits.json

```diff
@@ -44,10 +43,10 @@
             "Comparison Star(s) X & Y Pixel": [[465, 183], [512, 263]]
     },
     "planetary_parameters": {
-            "Target Star RA (hh:mm:ss)": "02:04:10",
-            "Target Star Dec (+/-hh:mm:ss)": "+46:41:23",
-            "Planet's Name": "HAT-P-32 b",
-            "Host Star's Name": "HAT-P-32",
+            "Target Star RA": "02:04:10",
+            "Target Star Dec": "+46:41:23",
+            "Planet Name": "HAT-P-32 b",
+            "Host Star Name": "HAT-P-32",
```

Comment:
Alternating this may cause an issue in exotic.py. I'd look at line 560 and see if you can update the key/value pair to go with these changes.

Reply:
Ah yea, forgot I changed that..This will need to be reverted to the original version
I was testing some dynamic programming and apostrophes don't play well in variable names.

Reply:
@rzellem Please use a different init file with today's release